### DISCUSSION TITLE
Using Lioness with Puma

### DISCUSSION THREAD

**Discussion by mazoilya** [netZoo/netZooPy#287] [Ideas]
I am wondering if there is a way to use Lioness with PUMA? I am looking for sample-specific microRNA regulators in my gene expression experiments. I read that in the R implementation Lioness can work with tools producing networks in a generic adjacency matrix. But then PUMA is available only on Python. It would be great if Lioness object can be constructed from PUMA object similar to how it's done with Panda objects in Python.

**Comment by marouenbg** [comment]
Hi @mararie, can you please have a look at this question? Thank you !

**Comment by marouenbg** [comment]
Hi @mazoilya,

I think this would be the function that you're looking for
https://github.com/netZoo/netZooPy/blob/master/netZooPy/lioness/lioness_for_puma.py

Let me know if this helps
